docs/pages/theming/simple.mdx

```diff
@@ -209,5 +209,5 @@ const theme = {
 This API makes super easy to add or remove button sizes or variants in the
 future.
 
-> All Chakra's component follows this API and we've made call component styles
+> All Chakra components use this API and we've made all component styles
```

Comment:
Is this part of the theme generation command you've built? We should probably document more what this means.